crates/bevy_app/src/app.rs

```diff
@@ -853,14 +853,6 @@ impl App {
         self
     }
 
-    /// Retrieves a "sub app" stored inside this [App]. This will panic if the sub app does not exist.
-    pub fn sub_app_mut(&mut self, label: impl AppLabel) -> &mut App {
-        match self.get_sub_app_mut(label) {
-            Ok(app) => app,
-            Err(label) => panic!("Sub-App with label '{:?}' does not exist", label),
-        }
-    }
```

Comment:
Why did you remove these functions?

Reply:
To force users handle the error. Was suggested by cart in https://github.com/bevyengine/bevy/issues/3155#issuecomment-987278686

Reply:
I put panicking API back.